.github/renovate.json5

```diff
@@ -80,16 +84,21 @@
     {
       customType: 'regex',
       fileMatch: [
-        '^Earthfile$',
+        'Earthfile$',
+        'earthfile2llb/with_docker_run_base\\.go',
+        'ast/tests/.*?\\.ast\\.json',
+        '.+.md',
       ],
       matchStrings: [
-        '# renovate: datasource=repology depName=alpine_(?<currentValue>\\d+_\\d+)',
-      ],
-      currentValueTemplate: "{{{ replace '_' '.' currentValue }}}",
+        'ARG(\\s+|\\s+.*?\\s+)DIND_IMAGE=earthly/dind:(?<currentValue>.*?)($|\\s|\\n)',
+        '--DIND_IMG=earthly/dind:(?<currentValue>.*?)($|\\s|\\n|")',
+        '("|\\`|\\s+)earthly/dind:(?<currentValue>.+?)("|\\`|\\s+\\|)',
+        'FROM(\\s+|\\s+.*?\\s+)earthly/dind:(?<currentValue>.*?)($|\\s|\\n|\\`)',
+        'docker.io/earthly/dind:(?<currentValue>.*?)@',
+
```

Comment:
nit: empty line

Reply:
thanks! I'll remove in a separate PR (too impatient to wait for the tests to pass again 😆 )

Reply:
totally fair.